apps/account/tests/test_views/test_guest_send_invitation_email_view.py

```
@@ -1,66 +1,67 @@
+
+
+def test_get_regular(authenticated_client, guest_user, room, guest_send_invitation_url):
+    response = authenticated_client.get(guest_send_invitation_url)
+
+    assert response.status_code == http.HTTPStatus.OK
+    assert response.template_name[0] == GuestSendInvitationEmailView.template_name
+    assert f'Invite {guest_user.name} to "{room.name}"' in response.content.decode()
+    assert response.context_data["active_tab"] == "people"
+
+
+def test_post_regular(authenticated_client, room, guest_user, guest_send_invitation_url, monkeypatch):
```

Ruff (0.14.7)

Unused function argument: `room`

(ARG001)